e2e/cypress/e2e/settings/private-labeling.cy.ts

```diff
@@ -40,7 +40,7 @@ function customize(theme: string, user: User) {
         cy.get('[data-e2e="image-part-logo"]')
           .find('input')
           .then(function (el) {
-            const blob = Cypress.Blob.base64StringToBlob(this.logo, 'image/png');
+            const blob = Cypress.Blob.base64StringToBlob(ctx.logo, 'image/png');
```

Comment:
vscode show the following error on line 1:
`Module '"../../support/api/apiauth"' declares 'API' locally, but it is not exported.`

Reply:
done